helm/hpcc/templates/network.yaml

```diff
@@ -0,0 +1,71 @@
+---
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: default-deny-all
+spec:
+  podSelector: {}
+  policyTypes:
+  - Ingress
+  - Egress
+  egress:
```

Comment:
You need to allow egress to kube-system for dns lookup to work, However this method is not very reliable, as it relies on the label having been set on the kube-system namespace which it is not by default.

Alternatives are:

1. Don't add egress controls
2. Allow egress to any machine on port 53 (dns)